transaction/extra/src/memo/mod.rs

```diff
@@ -415,4 +419,30 @@ mod tests {
         assert_eq!(memo.get_fee(), 17u64);
         assert_eq!(memo.get_num_recipients(), 4);
     }
+
+    #[test]
+    fn test_defragmentation_memo() {
```

Comment:
:+1: :+1: Really like these tests:
- Testing the boundary condition of max fee
- Testing the getters and setters, even though their straightforward the tests aren't that costly to write.
- Testing the error out of `new()`
- Testing serialzing

:+1: